### PR TITLE
SCA with Php Inspections (EA Extended)

### PR DESCRIPTION
**Pull request by kalessil**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

**Comment by kalessil** [comment]
Note: dropped assignments are redundant as default values matches the written values.

**Comment by nicolas-grekas** [comment]
Thank you @kalessil.

File: src/Symfony/Component/Intl/DateFormatter/DateFormat/TimeZoneTransformer.php [hunk 1]
```diff
@@ -94,6 +94,6 @@ public static function getEtcTimeZoneId($formattedTimeZone)
             return 'Etc/GMT'.($hours !== 0 ? $signal.$hours : '');
         }
 
-        throw new \InvalidArgumentException('The GMT time zone \'%s\' does not match with the supported formats GMT[+-]HH:MM or GMT[+-]HHMM.');
+        throw new \InvalidArgumentException(sprintf("The GMT time zone '%s' does not match with the supported formats GMT[+-]HH:MM or GMT[+-]HHMM.", $formattedTimeZone));
```
Comment (fabpot) [inline src/Symfony/Component/Intl/DateFormatter/DateFormat/TimeZoneTransformer.php]: We are using `"%s%"` in error message instead of `'%s'`

Comment (fabpot) [inline src/Symfony/Component/Intl/DateFormatter/DateFormat/TimeZoneTransformer.php]: fixed in fd94048285

**Pull request merged**